deploy.py

```diff
@@ -0,0 +1,171 @@
+import os
+import requests
+import configparser
+from ftplib import FTP, FTP_TLS
+
+
+class bcolors:
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
```

Comment:
You can do `print_blue`, `print_green`, etc... functions for easier use. For example:
```python
from functools import partiel
def printc(color: str, text: str):
    print(getattr(bcolors, color) + text + colors.ENDC
print_blue = partial(printc, color='OKBLUE')
```